bin/subprojects-install

```diff
@@ -72,6 +72,35 @@ done
 no_clone=""
 pushd "$crucible_repo_dir" >/dev/null || exit_error "Could not change directory to $crucible_repo_dir"
 /bin/mkdir -p subprojects/benchmarks subprojects/tools subprojects/core
+for link in $(find subprojects -type l); do
+    type_dir=$(echo "${link}" | awk -F'/' '{ print $2 }')
+    name_link=$(echo "${link}" | awk -F'/' '{ print $3 }')
+    active=0
+    if [ -n "${subprojects[$name_link]}" ]; then
+        sp_type=`echo ${subprojects[$name_link]} | awk '{print $2}'`
+        case "$sp_type" in
+            "benchmark")
+                if [ "${type_dir}" == "benchmarks" ]; then
+                    active=1
+                fi
+                ;;
+            "tool")
```

Comment:
wondering why "benchmark" and "tool" without 's' here and in the directory it is with 's'
```
subprojects/benchmarks/hwlatdetect
subprojects/benchmarks/cyclictest
subprojects/benchmarks/uperf
subprojects/benchmarks/tracer
subprojects/benchmarks/trafficgen
subprojects/benchmarks/oslat
subprojects/benchmarks/fio
subprojects/tools/ovs
subprojects/tools/tcpdump
subprojects/tools/ftrace
subprojects/tools/procstat
subprojects/tools/kernel
subprojects/tools/sysstat
subprojects/core/multiplex
subprojects/core/packrat
subprojects/core/workshop
subprojects/core/junk-drawer
subprojects/core/toolbox
subprojects/core/roadblock
subprojects/core/CommonDataModel
subprojects/core/rickshaw
```

Reply:
I agree that it does seem "odd", but its actually an inversion of the existing logic:

https://github.com/perftool-incubator/crucible/blob/master/bin/subprojects-install#L95-L105

Reply:
which I guess are influenced with the format that was chosen for this file originally:

https://github.com/perftool-incubator/crucible/blob/master/config/default_subprojects